Analysis/DataStorageProjects/NetappReplication.md

```diff
@@ -0,0 +1,114 @@
+
+The first time you invoke SnapMirror, it performs a baseline transfer from the source volume to the destination volume. The baseline transfer typically involves the following steps:
+
+* Make a Snapshot copy of the source volume.
+
+* Transfer the Snapshot copy and all the data blocks it references to the destination volume.
+
+* Transfer the remaining, less recent Snapshot copies on the source volume to the destination volume for use in case the active mirror is corrupted.
+
+Once a baseline transfer is complete, SnapMirror transfers only new Snapshot copies to the mirror. Updates are asynchronous, following the schedule you configure. Retention mirrors the Snapshot policy on the source. You can activate the destination volume with minimal disruption in case of a disaster at the primary site, and reactivate the source volume when service is restored.
+
+Because SnapMirror transfers only Snapshot copies after the baseline is created, replication is fast and nondisruptive. As the failover use case implies, the controllers on the secondary system should be equivalent or nearly equivalent to the controllers on the primary system to serve data efficiently from mirrored storage.
+
+![](netapp-replication.gif)
+
+#### Using SnapMirror for data transfer
+
+You can also use SnapMirror to replicate data between endpoints in the NetApp data fabric. You can choose between one-time replication or recurring replication when you create the SnapMirror policy.
+
+### The Implementation approach to achieve replication in netapp
+
+#### CreateReplication:
+
+##### 1. snapmirror create
+
+  ##### Note:
+  Before using this command to create a volume SnapMirror relationship, make sure the source volume should be in the online state and a read-write (RW) type and the destination volume should be in the online state and a data protection (DP) type. And also destination volume size should be greater equal than source volume size.
+
+
+  To create a synchronous SnapMirror relationship between the source Flexvol vs1.example.com:vol_log, and the destination Flexvol vs2.example.com:vol_log_sync_dp when the source cluster is running ONTAP 9.5 or above, type the following command:
+
+  * ##### snapmirror create -destination-path vs2.example.com:vol_log_sync_dp -source-path vs1.example.com:vol_log -policy Sync
+
+##### 2. snapmirror initialize:
+* Start a baseline transfer
+
+* The initial transfer to an empty destination volume is called a baseline transfer. During a baseline transfer for a data protection (DP) or extended data protection (XDP) relationship, the snapmirror initialize command takes a Snapshot copy on the source volume to capture the current image of the source volume. For data protection relationships, the snapmirror initialize command transfers all of the Snapshot copies up to and including the Snapshot copy created by it from the source volume to the destination volume.
+
+* ##### snapmirror initialize -destination-path vs2.example.com:vol_log_sync_dp
+
```

Comment:
Few comments:
1. We need to provide an overall architecture and how this connects to SODA projects
2. The interfaces and sequence diagram w.r.t soda. 
3.  please add the analysis conclusion or recommendations
If we are planning the above in our design separately, please add the links.